daprdocs/content/en/operations/hosting/kubernetes/kubernetes-overview.md

```diff
@@ -32,20 +41,29 @@ Deploying and running a Dapr enabled application into your Kubernetes cluster is
     dapr.io/config: "tracing"
 ```
 
+For more information, check [Dapr annotations]({{< ref arguments-annotations-overview.md >}}).
+
 ## Pulling container images from private registries
 
-Dapr works seamlessly with any user application container image, regardless of its origin. Simply init Dapr and add the [Dapr annotations]({{< ref arguments-annotations-overview.md >}}) to your Kubernetes definition to add the Dapr sidecar.
+Dapr works seamlessly with any user application container image, regardless of its origin. Simply [initialize Dapr]({{< ref install-dapr-selfhost.md >}}) and add the [Dapr annotations]({{< ref arguments-annotations-overview.md >}}) to your Kubernetes definition to add the Dapr sidecar.
 
 The Dapr control-plane and sidecar images come from the [daprio Docker Hub](https://hub.docker.com/u/daprio) container registry, which is a public registry.
 
-For information about pulling your application images from a private registry, reference the [Kubernetes documentation](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/). If you are using Azure Container Registry with Azure Kubernetes Service, reference the [AKS documentation](https://docs.microsoft.com/azure/aks/cluster-container-registry-integration).
+For information about:
+- Pulling your application images from a private registry, reference the [official Kubernetes documentation](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/). 
+- Using Azure Container Registry with Azure Kubernetes Service, reference the [AKS documentation](https://docs.microsoft.com/azure/aks/cluster-container-registry-integration).
 
-## Quickstart
+## Stopping the cluster in development
```

Comment:
You can remove this, since this addressed with the Dapr Injector watchdog that can be enabled on a local development cluster. I

Reply:
I meant you can remove this whole paragraph you added, not just the heading